simcon_project/conversation_templates/migrations/0003_auto_20210103_0336.py

```diff
@@ -0,0 +1,112 @@
+# Generated by Django 3.1.3 on 2021-01-03 03:36
```

Comment:
please delete this from the PR, i think u still dont have all the new main since this is in the git ignore